src/subnet_rental_canister/tests/integration_tests.rs

```diff
@@ -556,29 +493,15 @@ fn test_accept_rental_agreement_cannot_be_called_by_non_governance() {
         .contains(&format!("{:?}", ExecuteProposalError::UnauthorizedCaller)));
 }
 
+// TODO
+
+// fn test_proposal_rejected_if_already_rented() {
+// fn test_proposal_rejected_if_too_low_funds() {
```

Comment:
This is already tested with 50% of the initial payment in test_failed_initial_proposal